projects/core/src/i18n/i18next/i18next-init.ts

```diff
@@ -96,11 +99,21 @@ export class SiteContextI18nextSynchronizer implements OnDestroy {
  */
 export function i18nextGetHttpClient(
   httpClient: HttpClient
-): (url: string, options: object, callback: Function, data: object) => void {
-  return (url: string, _options: object, callback: Function, _data: object) => {
+): (
+  options: BackendOptions,
+  url: string,
+  payload: object | string,
+  callback: RequestCallback
+) => void {
+  return (
+    _options: BackendOptions,
+    url: string,
+    _payload: object | string,
+    callback: RequestCallback
+  ) => {
     httpClient.get(url, { responseType: 'text' }).subscribe(
-      (data) => callback(data, { status: 200 }),
-      (error) => callback(null, { status: error.status })
+      (data) => callback(null, { status: 200, data }),
+      (error) => callback(error, { data: error, status: error.status })
```

Comment:
Why do we pass the error as the `data`? This property is meant to be valid translation object.
See the original source code of i18next-http-backend:
https://github.com/i18next/i18next-http-backend/blob/217f0306da941e93c6e904974a2c66eae62921ab/lib/index.js#L67-L88

At the same time, I understand it's a typing issue in `i18next-http-backend`, that `data` is a required field in `RequestCallback`. However, I'm in favor of forcing `data` being undefined here.

Reply:
Good catch.
I'll just pass `null`, as this is what their docs say: 

```text
'err' should be an error
  //            'res' should be an object with a 'status' property and a 'data' property containing a stringified object instance beeing the key:value translation pairs for the
  //            requested language and namespace, or null in case of an error.

```